test/component-tests/envelopes.spec.js

```diff
@@ -134,6 +134,23 @@ test(function getRecipients (t) {
   });
 });
 
+test(function getRecipientsWithTabs (t) {
+  return client.envelopes.getRecipients(envelopeId, true)
```

Comment:
In an effort to make our tests self-documenting, please define a variable describing what the `true` value represents here.
```javascript
let include_tabs = true;
```